examples/tutorials/advance_model_config/01_microbatch.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>These operations are completely unclear from context of the tutorial. Also, <code>affect model initialization time</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Existence of these operations come from the description of microbatch.</p>